tests/CppUTestExt/MockSupportTest.cpp

```diff
@@ -697,6 +717,21 @@ TEST(MockSupportTest, outputParameterSucceeds)
     CHECK_NO_MOCK_FAILURE();
 }
 
+TEST(MockSupportTest, customObjectOutputParameterSucceeds)
```

Comment:
The additional test cases are no longer  needed when we just add a `copy()` method to the MockNamedValueComparator class.